distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -308,7 +307,7 @@ public void setup() throws Exception {
       fail("An unrecoverable error occurred from previous test");
     }
 
-    server = new SecureStubServer(Integer.parseInt(RESTITO_STUB_SERVER_PORT.getPort())).run();
+    server = new StubServer(Integer.parseInt(RESTITO_STUB_SERVER_PORT.getPort())).run();
```

Comment:
❓  was SecureStubServer no longer needed because of the upgrade?

Reply:
It was not. Yeats submitted a PR to the project and it was merged. The version we upgraded to includes Yeats's change.